db/rules.csv

```diff
@@ -53,4 +53,6 @@ VispdatScoreThreeOrLess,VI-SPDAT score of 3 or less,have
 VispdatScoreFourToSeven,VI-SPDAT score of 4 to 7,have
 VispdatScoreEightOrMore,VI-SPDAT score of 8 or more,have
 EnrolledInHmisProject,Enrolled in specified HMIS Project,be
-ActiveInCohort,Active in specified Cohort,be
\ No newline at end of file
+ActiveInCohort,Active in specified Cohort,be
+RequiringWheelchairAccessibility,Requiring wheelchair accessibility,be

```

Comment:
These should read well. "Must be Requiring wheelchair accessibility" doesn't quite sound right to me.  How about "Must be wheelchair accessible", so this becomes `RequiringWheelchairAccessibility,Wheelchair accessible,be` and `RequiringGroundFloor,Ground floor accessible,be`